files/gdb_environments

```diff
@@ -0,0 +1 @@
+GDB_JAVA_OPTS=-Dgraphdb.home=/var/opt/graphdb/node -Dgraphdb.home.conf=/etc/graphdb -Dhttp.socket.keepalive=true -XX:MaxRAMPercentage=85.0 -XX:-UseCompressedOops"
```

Comment:
add an empty line at the end, otherwise you'll get a single line when the user data script runs.